### PR TITLE
Buildkite migration

### PR DESCRIPTION
**Pull request by uri-weisman**
### Summary of your changes
1. Migrate our release process from Jenkins to Buildkite.

### Screenshot/Data
Screenshot 2023-05-18 at 9 43 24

### Related Issues
- Related: https://github.com/elastic/security-team/issues/4708

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

**Review by oren-zohar — CHANGES_REQUESTED** [review]
Can you please make sure you remove any Jenkins reference from wherever possible? e.g.:
- .github/CODEOWNERS
- .editorconfig

**Comment by oren-zohar** [comment]
@uri-weisman can you please update our [release docs](https://github.com/elastic/security-team/tree/main/docs/cloud-security-posture-team/release) (Jenkins -> Buildkite)

**Comment by oren-zohar** [comment]
The scripts `./ci/scripts` should be the scripts in our CI pipelines:
- `integration-tests.sh` can be removed, we don't use it
- `validate-mocks.sh` and `connect_kind.sh` are not used by our CI, we can move them into `/scripts`

File: bin/hermit.hcl [hunk 1]
```diff
@@ -1,4 +1,4 @@
 env = {
-  "CLOUDBEAT_VERSION": "8.8.0",
+  "CLOUDBEAT_VERSION": "8.9.0",
```
[the remark this answers is not in the document]
Comment (uri-weisman) [reply bin/hermit.hcl:2]: - https://github.com/elastic/security-team/pull/6699

File: .buildkite/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Buildkite
+
+This README provides an overview of the Buildkite pipeline used to automate the build and publish process for Cloudbeat artifacts.
+
+## Artifacts
+
+The pipeline generates the following artifacts:
+
+- **dependencies-CLOUDBEAT_VERSION-WORKFLOW.csv**: This CSV file contains a list of dependencies for the specific version of Cloudbeat being built. It helps track the dependencies used in the build process.
```
Comment (oren-zohar) [inline .buildkite/README.md:9]:
```suggestion
- **dependencies-CLOUDBEAT_VERSION-WORKFLOW.csv**: This CSV file contains a list of dependencies for the specific Cloudbeat version being built. It helps track build dependencies.
```

**Review by oren-zohar — APPROVED** [review]
:rocket: :rocket: :rocket:

**Pull request merged**